env-files/torch/generic_torch.sh

```diff
@@ -38,34 +36,34 @@ if [ -d "${cDir}/$ENV_NAME" ];then
 
   source $ENV_NAME/bin/activate
 else
-  python -m venv $ENV_NAME
+  python3 -m venv $ENV_NAME
 
   # activate env
   source $ENV_NAME/bin/activate
 
   echo "$ENV_NAME environment is created in ${cDir}"
 fi
 
-pip install --no-cache-dir --upgrade pip
-
-pip install --no-cache-dir packaging wheel
+pip install --no-cache-dir --upgrade pip 
+pip install --no-cache-dir packaging wheel 
+pip install --no-cache-dir 'numpy<2.0.0' || exit 1
```

Comment:
I would add a comment explaining that we had to add this constraint on numpy to avoid clashes with deepspeed